getting_started/index.rst

```diff
@@ -21,18 +21,36 @@ current distro release is always preferred.
    build-guide/build-3rd-party-toolchain
    build-guide/build-with-zephyr
 
-Set up SOF on hardware
-**********************
+Set up SOF on a Linux machine
+*****************************
 
-SOF runs on a variety of devices with varying audio capabilities so
-instructions may differ between devices.
+You can build the Linux kernel with the latest SOF code and install it locally or remotely with ktest. 
+
+Do this first:
+
+.. toctree::
+   :maxdepth: 1
+
+   setup_linux/prepare_build_environment
```

Comment:
Where does `setup_linux` come from? First time I see this :-(

Reply:
I think it's just document structure?  IE: [getting_started/setup_linux/install_locally.rst](https://github.com/thesofproject/sof-docs/pull/404/files/4440d05adacc9ed228916f8b87657e817010ba51#diff-7a498d5cd8f9c6d9101facddd786e288b7e99e4351d638acf7296cd8f86b10b1)

Reply:
Doh! Thanks @greg-intel